lib/chef/knife.rb

```diff
@@ -182,7 +183,7 @@ def self.load_config(explicit_config_file, profile)
       config_loader.profile = profile
       config_loader.load
 
-      ui.warn("No knife configuration file found. See https://docs.chef.io/config_rb_knife.html for details.") if config_loader.no_config_found?
+      ui.warn("No config.rb (formerly #{Chef::Dist::KNIFE}.rb) configuration file found. See https://docs.chef.io/config_rb_knife.html for details.") if config_loader.no_config_found?
```

Comment:
Let's leave non-distro related changes out of this.

Reply:
how would we formulate it so it's relevant regardless of knife's name?

Reply:
TBH I still wonder why knife is included in chef and not it's own product, it's mostly irrelevant for node management, I probably don't know how much Chef and knife codebase are interleaved that said

Reply:
history, we could peel a knife gem out of the chef gem but it would have to dep on the chef gem and we'd ship it in the same repo and march versions in lockstep anyway, so the advantage would be fairly low.   we'd still ship it in the omnibus chef package, so it wouldn't reduce footprint any as well.  so, not a lot of practical impetus to break it out.

Reply:
Fair enough :)

Reply:
I don't see the need to change knife.rb so I think this can same the way it is. knife.rb is the legacy config name and everyone should be on config.rb going forward which eliminates the need to worry about the file name.